.github/workflows/haskell.yml

```diff
@@ -210,6 +210,4 @@ jobs:
             trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= loony-tools:pr9m4BkM/5/eSTZlkQyRt57Jz7OMBxNSUiMC4FkcNfk=
             substituters = https://cache.iog.io https://cache.nixos.org/ https://cache.zw3rk.com
       - name: nix-build
-        run: nix-build
-      - name: nix-shell
-        run: nix-shell shell.nix
+        run: nix build .#hydraJobs.required
```

Comment:
Why not also `nix develop` ? Is it not possible that this will fail if `nix build` doesn't?

Reply:
the required job does include the `nix develop` shell so it should cover `nix develop` case.
But thinking of it, this job could be more useful by checking that hydra or cicero di their job properly, by checking that everything is in cache already (no job should be built locally). Adding a commit (664ece0e47cf711d3af055f0b5d9f784a83aaead) for this.